.gitignore

```diff
@@ -8,6 +8,6 @@ npm-debug.*
 *.mobileprovision
 *.orig.*
 web-build/
-
+.package-lock.json/*
```

Comment:
Did @JethroGibbsN ask you to add this? [Please read this](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5)